community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,123 @@
 # Knative Steering Committee
 
-The Knative Steering Committee (SC) defines, evolves, and defends the vision,
-values, mission, and scope of the project. _The Steering Committee is a
-work-in-progress._
+The Knative Steering Committee (KSC) is the ultimate authority for the Knative
+project, and controls all aspects of the project or delegates its authority
+to other committees.
+
+KSC has delegated technical guidance for the project to the
+[Technical Oversight Committee](TECH-OVERSIGHT-COMMITTEE.md). However, the KSC
+acts as the final escalation point for any and all decisions within the project.
 
 - [Charter](#charter)
+- [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
+  - [Allocation of seats](#allocation-of-seats)
+- [Frequently asked questions](#frequently-asked-questions)
+- [Getting in touch](#getting-in-touch)
 
 ## Charter
 
-- Non-technical project oversight
+1. Define, evolve, and defend the vision, values, mission, and scope of the
+   project - to establish and maintain the soul of Knative.
+1. Define and evolve project governance structures and policies, including
+   project roles and how collaborators become members, approvers, leads,
+   and/or administrators. This includes policy for the creation and
+   administration of community groups, [working groups](WORKING-GROUPS.md) and
+   committees.
+1. Approve membership to the Technical Oversight Committee.
+1. Steward, control access, delegate access, and establishes processes regarding,
+   all Knative project resources and has the final say in the disposition of
+   those resources.
+1. Manage the Knative brand and decide which things can be called "Knative" and
+   how that mark can be used in relation to other efforts or vendors.
+1. Act as the final escalation point for any disputes or escalations from the
```

Comment:
I would add the followings responsibilities to the committee for now, until further process and groups are defined to cover these items:
1. All members would have access to the committee email, any email send to the comittee it's privacy ie maintain by the members.
1. Receive security reports via committee email and work with the appropriate technical leads to respond to the report as rejected or accepted, and oversee that the technical leads handle the report appropriately in private until the CVE is disclosed to the broader community.
1. Receive reports about code of conduct violations, and handle these reports and take actions protecting confidentiality.